### PR TITLE
Image Utility Fixes

### PR DESCRIPTION
**Pull request by EvanLovely**
Fixing base64 function call that resulted in math like 50 / '16,16' causing incorrect output

This is the source of the error: Notice: A non well formed numeric value encountered in /home/travis/build/bolt-design-system/bolt/vendor/gregwar/image/Gregwar/Image/Adapter/GD.php on line 77

### CODE REVIEW

**Comment by EvanLovely** [comment]
New deploy up

Built with commit 8e098ee3eee4a71134295f9ce288c14963913444

https://deploy-preview-488--bolt-design-system.netlify.com

File: packages/core-php/src/Images.php [hunk 1]
```diff
@@ -91,7 +91,7 @@ public static function calculate_average_image_color($relativeImagePath, $wwwDir
       }
 
       // Resize and optimize the image before running through ColorThief
-      $resizedImage = \Gregwar\Image\Image::open($absoluteImagePath)->resize('640,640')->jpeg($quality = 50);
+      $resizedImage = \Gregwar\Image\Image::open($absoluteImagePath)->resize('640', '640')->jpeg($quality = 50);
```
Comment (sghoweri) [inline packages/core-php/src/Images.php]: Wait - seriously?? That's the thing throwing all that noise when we're compiling in the Travis build?!

Comment (EvanLovely) [reply packages/core-php/src/Images.php]: Yep, resulted in this little gem of a discovery deep inside that `Image` class: 

screenshot_2018-02-24_05-29-10_pm

I've got my debugger on, so you can see the values; it was trying to do the math `50 / '16,16'` - which lead us to our mysterious PHP notice (not error - which is why it was not failing build):

> Notice: A non well formed numeric value encountered

**Comment by EvanLovely** [comment]
Images aren't all the way done yet though...

**Comment by sghoweri** [comment]
> Images aren't all the way done yet though... 

Even still, progress is progress. Really appreciate your help on this @EvanLovely!

**Comment by EvanLovely** [comment]
GOT IT!!! Everything's working that needs to! We definitely will chat soon; but the fundamentals are working!

**Comment by EvanLovely** [comment]
Now I'm missing icons on the home page of the bolt site 😢

**Comment by EvanLovely** [comment]
AWESOME! 🎉Fixed! Much to talk about, but this is working!!

**Pull request merged**